tests/test_bisnd_mcc.py

```diff
@@ -54,7 +54,7 @@ def test_status():
 
 def test_location():
     assert parsed_item["location"] == {
-        "address": "Morton County Commission Room",
+        "address": "Morton County Commission Room",  # noqa shouldn't "address" value contain the street/city/state instead of "name" value?
```

Comment:
The `name` and `address` is probably switched.

Reply:
gonna keep this comment for the real merge so other devs can see.